sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -233,6 +233,19 @@ typedef struct
   } _internal;
 } az_iot_hub_client_properties;
 
+/**
+ * @brief Supported IoT Hub properties
+ */
+static const az_span az_iot_hub_client_properties_message_id
+    = AZ_SPAN_LITERAL_FROM_STR("%24.mid"); /**< Add unique identification to a message */
+static const az_span az_iot_hub_client_properties_correlation_id = AZ_SPAN_LITERAL_FROM_STR(
+    "%24.cid"); /**< Used in distributed tracing. More information here:
+                   https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-distributed-tracing */
+static const az_span az_iot_hub_client_properties_content_type = AZ_SPAN_LITERAL_FROM_STR(

```

Comment:
```suggestion
static const az_span az_iot_hub_properties_content_type = AZ_SPAN_LITERAL_FROM_STR(

```

Reply:
these can be used for both hub client and messages.

Reply:
`az_iot_hub_message_properties_content_type` or `az_iot_hub_properties_content_type`

Reply:
the other's have a `_message_` in there

Reply:
The other has _message_ is because they can only be used on message properties. for content type and content encoding, can be used on both

Reply:
1. Are we sure the format would be the same for the connect.
2. The user name doesn't have a properties parameter so we wouldn't be able to pass it there.

Reply:
looking at the CBOR spec, the property is "format", so I was wrong.

`Username: {iothubhostname}/{device_id}/?api-version=2018-06-30[&format=json|cbor]. `

https://microsoft.sharepoint.com/:w:/r/teams/Azure_IoT/_layouts/15/Doc.aspx?sourcedoc=%7B927931C4-0C4A-4B02-8644-18D8F9C76D29%7D&file=Twin-protocol-exchange-optimization.docx&action=default&mobileredirect=true&CID=2325737B-62EF-4550-8483-424C7308DD24&wdLOR=c0EF02DD5-BA49-4BC6-98D4-BE53B12B921A

Reply:
note, CBOR support is out of scope for us right now